packages/api/src/schema/api-schema.yaml

```diff
@@ -1663,15 +1648,6 @@ components:
         disabled:
           type: boolean
           description: Disable the signing key to allow rotation safely
-    signing-key-response-payload:
```

Comment:
This is actually an API object. It should stay here and we should actually change https://github.com/livepeer/livepeer-com/blob/e98108915d4ead04eaec4ba75240084355422d00/packages/api/src/schema/api-schema.yaml#L3466 to reference it instead of the other one which is wrong.